### PR TITLE
Added forkedFromProject Json Object

### PR DESCRIPTION
**Pull request by aram535**

### CODE REVIEW

**Comment by EmilienMottet** [comment]
duplicate of #242

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,9 +1,28 @@
-# How to contribute
 
+This is a copy of Lukas Eder's JOOq Contributing.md guide - I'll be modiying it for this repository shortly.
+Credit for original goes to Lukas Eder's Jooq CONTRIBUTING.md file.  Original can be found at: https://raw.githubusercontent.com/jOOQ/jOOQ/master/CONTRIBUTING.md
 
-* The integration tests need a running Docker infrastructure.
```
Comment (timols) [inline CONTRIBUTING.md]: I'm not sure of the need to drastically change the contribution guide - would you mind elaborating on why?

Comment (aram535) [reply CONTRIBUTING.md]: Great! Are you back? 
My apologies, that should not have made it into this PR.  You can close this out as the main content is a duplicate of 242 anyway.

Comment (timols) [reply CONTRIBUTING.md]: Yes, sorry about that. Ok, will close this PR.

**Pull request closed without merging**